docs/spec/SPEC-SPEC.md

```diff
@@ -7,20 +7,30 @@ this directory.
 
 For consistency, specs should be written in passive present tense.
 
+## Pseudo-Code
+
+Pseudo-code should be avoided throughout the spec, instead bulleted-lists should 
+be used to describe with words all operations which occur. 
+
 ## Common Layout
 
 The following generalized structure should be used to breakdown specifications
-for modules. Note that not all files may be required depending on the modules 
-function
-
- - `overview.md` - describe module
- - `state.md` - specify and describe structures expected to marshalled into the store, and their keys
- - `state_transitions.md` - standard state transition operations triggered by hooks, messages, etc. 
- - `end_block.md` - specify any end-block operations
- - `begin_block.md` - specify any begin-block operations
- - `messages.md` - specify message structure and expected state machine behaviour 
- - `hooks.md` - describe available hooks to be called by/from this module 
- - `tags.md` - list and describe event tags used
+for modules. With the exception of README.md, `XX` at the beginning of the file
+name should be replaced with a number to indicate document flow (ex. read
+`01_state.md` before `02_state_transitions.md`). Note that not all files may be
```

Comment:
Generally, I can see how this will help. But I can certainly foresee some specs not needing to abide by this "flow" because there simply isn't one. Is this what you mean by:

`Note that not all files may be required depending on the modules function.`?

If so, I recommend changing it to:

`Note, not all files may be required to follow this convention depending on the module's function.`?

Reply:
your revision didn't quite capture the intention, updated to `The following list is nonbinding and 
all files are optional.`

Generally, most specs will naturally contain a flow based on information comprehension (important to understand state before state transitions for instance) - However I think flow _should_ be enforced even if it's artificial to the content of the files. This gives the possibility for easy exports (ex. pdf). And besides the files need to be listed in some order no matter - why not make that order explicit?

Reply:
Fair enough 👍